bundle/manifests/hawt.io_hawtios.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     kind: Hawtio
     listKind: HawtioList
     plural: hawtios
+    shortNames:
+    - hawt
```

Comment:
Do we need this?  Maybe we can consider other options: `hwt`, `hio`, etc. `hawt` doesn't seem to shorten so much.